YoutubeExplode/YoutubeClient.cs

```diff
@@ -404,10 +410,17 @@ public YoutubeClient()
 
                     // Fix for Hebrew
                     if (lang == "iw") lang = "he";
-
-                    var culture = new CultureInfo(lang);
-                    var caption = new ClosedCaptionTrackInfo(url, culture, isAuto);
-                    captions.Add(caption);
+                    // Fix for Yi
+                    if (lang == "yi") lang = "ii";
+
+                    try {
```

Comment:
[This](https://social.msdn.microsoft.com/Forums/vstudio/en-US/70f01bf8-290b-43e3-a2ae-08870220c3c8/cultureinfogetcultureinfo-behavior-change-on-windows-10?forum=netfxbcl) states that it only will create such an `und`-object on windows 10 (or later, I guess) which I can confirm. It makes sense because the `und`-language isn't specified on earlier systems.

I first wanted to check if the current culture exists on the current system and then proceed depending on that, but `CultureInfo.GetCultures` is not in .Net Standard ([which also makes sense](https://github.com/dotnet/corefx/issues/1669)). That's why I've gone with try-catch.